tsconfig.json

```diff
@@ -4,6 +4,6 @@
   "compilerOptions": {
     "outDir": "./build"
   },
-  "include": ["./src"],
+  "include": ["./src", "example/components"],
```

Comment:
I think we should move `components` to a separate place, if we need to use them later in our real project, and not put them into `example`

Reply:
just a temporary, we will remove when packages repo is ready.